src/wallet/scriptpubkeyman.cpp

```diff
@@ -1822,7 +1822,7 @@ std::optional<MigrationData> LegacyScriptPubKeyMan::MigrateToDescriptor()
 
             // Make the combo descriptor
             std::string xpub = EncodeExtPubKey(master_key.Neuter());
-            std::string desc_str = "combo(" + xpub + "/0'/" + ToString(i) + "'/*')";
+            std::string desc_str = "combo(" + xpub + "/0h/" + ToString(i) + "h/*')";
```

Comment:
You are missing one occurrence in this expression:
```suggestion
            std::string desc_str = "combo(" + xpub + "/0h/" + ToString(i) + "h/*h)";
```